client/src/component/Title/Title.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import 'font-awesome/css/font-awesome.min.css'
+import './Title.css'
+
+const Title = () => {
+    return (
+        <article className="Title" id="Header">
+            <section className="container">
+                <header>HACKMERCED V</header>
```

Comment:
See if you can create two different font sizes for the header and the date. Try the header being the biggest font on that page, bold, and in your face, see how that works out.

Reply:
```suggestion
                <h1>HACKMERCED V</h1>
                <h6>20/20/2020</h6>
```

you can use the `<h1>`, `<h2>`, etc tags